metadata-ingestion/tests/unit/test_bigquery_usage.py

```diff
@@ -787,8 +842,10 @@ def test_usage_counts_no_columns(
         ),
     ]
     with caplog.at_level(logging.WARNING):
-        workunits = usage_extractor._run(events, TABLE_REFS.values())
-        assert list(workunits) == [
+        workunits = usage_extractor._get_workunits_internal(
+            events, [TABLE_REFS[TABLE_1.name]]
+        )
```

Comment:
Curious why this changed from `TABLE_REFS.values()` to `[TABLE_REFS[TABLE_1.name]]` ?

Reply:
The refs are used to determine which entities need zero usage, and I wanted to keep this test simple (i.e. smaller `expected`)